fedimint-server/src/consensus/server.rs

```diff
@@ -323,29 +323,30 @@ impl ConsensusServer {
     }
 
     pub async fn run_session(&self, session_index: u64) -> anyhow::Result<()> {
-        // this constant needs to be 3000 or less to guarantee that the session
-        // can never reach MAX_ROUNDs.
-        let exponential_slowdown_offset: usize =
-            3 * self.cfg.consensus.broadcast_expected_rounds_per_session as usize;
-        let max_round = self.cfg.consensus.broadcast_max_rounds_per_session;
-        let round_delay = self.cfg.local.broadcast_round_delay_ms as f64;
-        const BASE: f64 = 1.01;
-
-        // this is the minimum number of unit data that will be ordered before we reach
-        // the EXPONENTIAL_SLOWDOWN_OFFSET even if f peers do not attach unit data
-        let batches_per_session = exponential_slowdown_offset * self.keychain.peer_count();
-
         // In order to bound a sessions RAM consumption we need to bound its number of
         // units and therefore its number of rounds. Since we use a session to
         // create a threshold signature for the corresponding session outcome we have to
         // guarantee that an attacker cannot exhaust our memory by preventing the
         // creation of a threshold signature, thereby keeping the session open
-        // indefinitely. Hence we increase the delay between rounds exponentially
-        // such that MAX_ROUND would only be reached after roughly 350 years.
+        // indefinitely. Hence, we increase the delay between rounds exponentially
+        // such that max_round would only be reached after a minimum of 100 years.
```

Comment:
That is enforced by AlephBFT on startup, right?

Reply:
Yes